cfme/rest/gen_data.py

```diff
@@ -593,7 +592,10 @@ def policies(request, rest_api, num=2):
 
 
 def get_dialog_service_name(appliance, service_request, *item_names):
-    """Helper to DRY this VersionPicker when tests need to determine a dialog service name
+    """Note: VersionPicker has currently been removed,
+    since None value is returned in the service_request options.
```

Comment:
This `docstring` will create confusion :).  Parthvi: now, if for all version needs to parse message then please remove this. Else please explain your thought.